bin/retrieve_dns.py

```diff
@@ -301,9 +307,6 @@ def runprocess(config_file, log_config_file):
             new_dn_file.write(dn)
             new_dn_file.write('\n')
             added += 1
-        elif dn.lstrip().startswith("#"):
```

Comment:
why are these lines being removed?

Reply:
comments are now filter out from the DN files at read time, and GOCDB won't allow `#` symbols in the relevent field.